global-api/routes/city_boundaries_endpoint.py

```diff
@@ -1,109 +1,53 @@
 from db.database import SessionLocal
 from fastapi import HTTPException, APIRouter
-from sqlalchemy.orm import sessionmaker
-from sqlalchemy import text, and_
-import json
-from models.osm import Osm
+from sqlalchemy import text
 from decimal import Decimal
-from shapely.geometry import Point, shape, Polygon, MultiPolygon
-from shapely.wkt import loads
-from functools import partial
-from shapely.ops import transform
-import math
-from pyproj import Transformer
 
 api_router = APIRouter(prefix="/api/v0")
 
-
 # Extract the polygon by locode
 def db_query(locode):
     """
-        Query the database to retrieve OSM data for a given locode.
+        Query the database to retrieve city polygon data for a given locode.
 
         Args:
             locode (str): The location code to filter by.
 
         Returns:
-            Osm: The queried row from the OSM table matching the locode.
+            dict: The queried row from the modelled.city_polygon table matching the locode.
+            None: If no city is found with the given locode.
```

Comment:
Since the output schema was apparently changed, do we need to create a new version of this endpoint for compatibility?